style.css

```diff
@@ -0,0 +1,854 @@
+  }
+
+  .navbar-links > li > a {
+    text-decoration: none;
+  }
+
+  .small-container {
+    margin-top: 5.5%;
+    background: url(./images-portfolio-desktop/Header\ bg.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-color: white;
+    height: 100vh;
+    padding: 100px 80px;
+  }
+
+  h1 {
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    margin-bottom: 2px;
+  }
+
+  .skills > span > img {
+    position: relative;
+    top: 10px;
+    left: -80px;
+  }
+
+  .card-info {
+    padding: 0 1%;
+    grid-template: repeat(4, auto) / minmax(0, 1fr);
+  }
+
+  .card-item {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    padding: 16px;
+    padding-bottom: 22px;
+  }
+
+  .images-tags {
+    display: block;
+    max-width: 100%;
+    width: 100%;
+  }
+
+  .images-tags > a > img {
+    max-width: 100%;
+    display: block;
+    height: auto;
+    object-fit: contain;
+  }
+
+  .card-item:nth-child(even) {
+    flex-direction: row-reverse;
+  }
+
+  .main-text {
+    display: flex;
+    flex-direction: column;
+    padding-inline-start: 18px;
+  }
+
+  .title {
+    font-size: 2rem;
+    line-height: 2rem;
+    color: #091e42;
+    margin-bottom: 0;
+  }
+
+  .title2,
+  .title3 {
+    visibility: hidden;
+    margin-bottom: -4vh;
+  }
+
+  .title2::after,
+  .title3::after {
+    content: "Facebook 360";
+    visibility: visible;
+    display: block;
+    position: relative;
+    top: -2vh;
+    left: 0;
+  }
+
+  .title3::after {
+    content: "Uber Navigation";
+  }
+
+  .description {
+    font-size: 1em;
+    line-height: 1rem;
+    padding-right: 60px;
+  }
+
+  .frame-2 {
+    gap: 2%;
+    align-items: center;
+  }
+
+  .frame-end {
+    margin-left: -6%;
+  }
+
+  .RoR {
+    display: inline-block;
+  }
+
+  .top-title,
+  .details {
+    font-size: 1em;
+    line-height: 1em;
+    color: #344563;
+  }
+
+  .details {
+    font-weight: 400;
+    color: #6b778c;
+  }
+
+  .top-title2,
+  .top-title4 {
+    visibility: hidden;
+  }
+
+  .top-title2::after,
+  .top-title4::after {
+    visibility: visible;
+    display: block;
+    position: relative;
+    top: -1.5vh;
+    left: 0;
+    content: "FACEBOOK";
+  }
+
+  .top-title4::after {
+    content: "UBER";
+    padding-inline-start: 30%;
+  }
+
+  .stack,
+  .end-stack {
+    visibility: hidden;
+  }
+
+  .stack::after,
+  .end-stack::after {
+    visibility: visible;
+    display: block;
+    position: relative;
+    top: -1.5vh;
+    left: 0;
+    content: "Full Stack Dev";
+  }
+
+  .end-stack::after {
+    content: "Lead Developer";
+  }
+
+  .tags {
+    gap: 2%;
+  }
+
+  .more-button {
+    width: 130px;
+    height: 48px;
+  }
+
+  .about {
+    background: url(./images-portfolio-desktop/about\ myself\ -\ background-\ shape-\ desktop@2x.png);
+    flex-direction: row;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: right;
+    padding: 8% 2%;
```

Comment:
- [ ] :thinking: Kindly make sure there si a continuity of the white background color as it is shown on the Figma design:

Your design:
![Screenshot from 2023-01-18 19-08-05](https://user-images.githubusercontent.com/97834160/213247919-7b0d77e9-8fd6-4a14-8491-1dcd3fc565ea.png)

Figma:
![Screenshot from 2023-01-18 18-54-57](https://user-images.githubusercontent.com/97834160/213247578-61d56969-2127-461b-8575-36948eabce6b.png)